### PR TITLE
Lesson 3 add files

### PR DESCRIPTION
**Pull request by GrzegorzBajorski**
Signed-off-by: Grzegorz Bajorski <Grzegorz.Bajorski@tieto.com>

### CODE REVIEW

**Comment by GrzegorzBajorski** [comment]
@gradopaw please review

File: students/grzegorz_bajorski/lesson_03_functions/input_validation.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+number = 0
+
+while 1:
+    try:
+        number = int(input())
+        if collatz(number) != 1:
+            print(collatz(number))
+        else:
+            break
+    except:
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_03_functions/input_validation.py]: Too broad exception clauses. When you catching exceptions you should mention specific exceptions whenever it is possible. Here as it is suggested in the task use ValueError.

File: students/grzegorz_bajorski/lesson_03_functions/the_collatz_seq.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+def collatz(number):
```
Comment (gradopaw) [inline students/grzegorz_bajorski/lesson_03_functions/the_collatz_seq.py]: Please have a look at comments to input_validation.py and fix them here as well.

**Comment by gradopaw** [comment]
@GrzegorzBajorski are you going fix my comments?

**Comment by GrzegorzBajorski** [comment]
Yes, I will fix it

**Pull request merged**